test/functional/rpc_rawtransaction.py

```diff
@@ -323,5 +324,29 @@ def run_test(self):
         decrawtx= self.nodes[0].decoderawtransaction(rawtx)
         assert_equal(decrawtx['vin'][0]['sequence'], 4294967294)
 
+        ####################################
+        # TRANSACTION VERSION NUMBER TESTS #
+        ####################################
+
+        # Use Mainnet transaction c659729a7fea5071361c2c1a68551ca2bf77679b27086cc415adeeb03852e369
```

Comment:
I don't see the reason for using this particular transaction. The max and min versions below seem sufficient to me.

Reply:
Thanks @jnewbery,  I have removed the test vector per your suggestion.